arcgis-ios-sdk-samples/Layers/Group layers/GroupLayersSectionView.xib

```diff
@@ -0,0 +1,50 @@
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="LayersTableViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="XjP-9q-BD7" customClass="GroupLayersSectionView" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="44"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" horizontalCompressionResistancePriority="749" text="Label" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="10R-ek-huY">
+                    <rect key="frame" x="16" y="5" width="300" height="34"/>
+                    <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
+                    <nil key="textColor"/>
+                    <nil key="highlightedColor"/>
+                </label>
+                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="9ku-4w-xzb">
+                    <rect key="frame" x="321" y="5" width="51" height="34"/>
+                    <connections>
+                        <action selector="toggleSwitch:" destination="-2" eventType="valueChanged" id="ndi-hb-abW"/>
+                    </connections>
+                </switch>
+            </subviews>
+            <constraints>
+                <constraint firstItem="10R-ek-huY" firstAttribute="top" secondItem="XjP-9q-BD7" secondAttribute="top" constant="5" id="1AJ-iz-MjX"/>
+                <constraint firstItem="9ku-4w-xzb" firstAttribute="leading" secondItem="10R-ek-huY" secondAttribute="trailing" constant="5" id="Lt2-qq-1br"/>
+                <constraint firstItem="10R-ek-huY" firstAttribute="leading" secondItem="XjP-9q-BD7" secondAttribute="leading" constant="16" id="NKS-xX-JVd"/>
+                <constraint firstAttribute="bottom" secondItem="9ku-4w-xzb" secondAttribute="bottom" constant="5" id="QuA-ha-W9X"/>
+                <constraint firstAttribute="bottom" secondItem="10R-ek-huY" secondAttribute="bottom" constant="5" id="inj-lU-HBo"/>
+                <constraint firstItem="9ku-4w-xzb" firstAttribute="trailing" secondItem="XjP-9q-BD7" secondAttribute="trailing" constant="-5" id="w8s-53-RrP"/>
```

Comment:
Please constrain to the trailing margin of the superview:

![image](https://user-images.githubusercontent.com/2257493/56171339-bdc5c280-5f99-11e9-9a01-17700b7e7413.png)

Same with the table view cells.